api/v1beta1/containerjfr_types.go

```diff
@@ -53,7 +53,7 @@ type ContainerJFRSpec struct {
 	StorageOptions *StorageConfiguration `json:"storageOptions,omitempty"`
 	// Ingress specifications when running outside of OpenShift environment
 	// +optional
-	IngressOptions *IngressConfigurationList `json:"ingressOptions,omitempty"`
+	NetworkOptions *NetworkConfigurationList `json:"networkOptions,omitempty"`
```

Comment:
The comment here could use an update now. Something generic like "Options to control how the operator exposes the application over a network".

Reply:
Right, I forgot to make it less specific for the future!